### DISCUSSION TITLE
Unable to display the response from post request of form on page

### DISCUSSION THREAD

**Discussion by syedshadab19** [OrchardCMS/OrchardCore#11804] [General]
Hi,

I have created a page with a form as shown in below snapshot 1. when I click on submit button the form submits the data to a workflow shown in snapshot 2 which performs HTTP request I want to display the response data of the HTTP request on the page of the form shown in snapshot 1.

Snapshot 1 : 
![image](https://user-images.githubusercontent.com/25971751/171872023-137ed5ce-0b14-4399-8fb7-23b04a3df2ef.png)

Sanpshot 2:
![image](https://user-images.githubusercontent.com/25971751/171872160-c87a49ba-ba8a-4b8b-a453-8ae66d52d6fd.png)

**Comment by lampersky** [comment]
it would be better to create "new discussion" instead of issue, please have a look on my [workflow examples](https://github.com/lampersky/OrchardCoreWorkflowsExamples), if this is not enough, and you really found an issue let us know

**Reply by syedshadab19** [reply]
Ok will check the workflow examples.

**Reply by syedshadab19** [reply]
Hi checked the samples I am unable to display the response in same page. the examples create a new page using workflow to display the response but I want it to displayed the same page it self
